### PR TITLE
fix for https://github.com/jtesta/ssh-audit/issues/280

### PR DESCRIPTION
**Pull request by dreizehnutters**
This PR is fixes the issue https://github.com/jtesta/ssh-audit/issues/280 in order to make the json format more consistant.

grafik

### CODE REVIEW

**Comment by jtesta** [comment]
@dreizehnutters : in case you didn't see it, I replied with some feedback on your proposed change.

**Comment by dreizehnutters** [comment]
hey @jtesta where exactly did you post the feedback ? I cant find it ?!

**Comment by jtesta** [comment]
Oops.  That was my fault.  I started a review, but left it as a draft.  I just submitted it; you should be able to see it now.

File: src/ssh_audit/ssh_audit.py [hunk 1]
```diff
@@ -735,7 +735,7 @@ def evaluate_policy(out: OutputBuffer, aconf: AuditConf, banner: Optional['Banne
 
     passed, error_struct, error_str = aconf.policy.evaluate(banner, kex)
     if aconf.json:
-        json_struct = {'host': aconf.host, 'policy': aconf.policy.get_name_and_version(), 'passed': passed, 'errors': error_struct}
+        json_struct = {'target': aconf.host + ":" + str(aconf.port), 'policy': aconf.policy.get_name_and_version(), 'passed': passed, 'errors': error_struct}
```
[the remark this answers is not in the document]
Comment (dreizehnutters) [reply src/ssh_audit/ssh_audit.py:738]: That is thoughful, I will change the PR accordingly.

**Comment by dreizehnutters** [comment]
ready for review again

**Comment by jtesta** [comment]
Merged.  Thanks!

**Pull request merged**